src/client/metadataTransfer.ts

```diff
@@ -115,14 +115,26 @@ export abstract class MetadataTransfer<
       }
       return result;
     } catch (e) {
```

Comment:
this might be better as an exported function (pass in the `e` and the `id`) so that we could have a UT that imports it and throws various errors at it (regular Error, SfError, an `isogit` error) and asserts that we're giving back SfErrors with the correct input.